### ISSUE TITLE
logits as a result

### ISSUE THREAD

**Issue by sagol** [kaushaltrivedi/fast-bert#10]
What do we need to specify for the labels if we need logits as a result?
